fractions/wildfly/elytron/src/main/java/org/wildfly/swarm/elytron/ElytronFraction.java

```diff
@@ -80,11 +79,6 @@ public ElytronFraction applyDefaults() {
             providers.provider(ELYTRON);
             providers.provider(OPENSSL);
         });
-        fileAuditLog(LOCAL_AUDIT, (log) -> {
-            log.path("audit.log");
-            log.relativeTo("jboss.server.log.dir");
-            log.format(Format.JSON);
-        });
```

Comment:
In place of these 4 removed lines, could you add a comment with a link to the JIRA? That way, when someone in the future compares the default Thorntail and WildFly configuration, they will know this is intentional.